tests/contrib/django/test_django.py

```diff
@@ -377,6 +377,24 @@ def test_lambda_based_view(client, test_spans):
         assert span.resource == "GET tests.contrib.django.views.<lambda>"
 
 
+def test_template_view(client, test_spans):
```

Comment:
Are we able to add a test that has `django.response.template` as a list?

Reply:
will do 👍